backend/RayTracing/Classes/Models/Vector.py

```diff
@@ -69,5 +69,16 @@ def getNegative(self):
         v = Vector(- self.x, - self.y, - self.z)
         return v
 
-    def getArray(self):
-        return [self.x, self.y, self.z]
+    def __add__(self, other):
```

Comment:
I think these methods are unnecessary or?

Reply:
Yeah, you are right, I removed them. Although, this seems like a good way to implement these functions, is there a specific reason we did not go with that from the beginning?

Reply:
No there is no specific reason, but we already have this methods. If you prefer this naming convention we can keep them and refactor the code :)

Reply:
No, I think it would be a pain to refactor them. This is fine as well, I was just wondering.